app/controllers/links_controller.rb

```diff
@@ -54,7 +54,8 @@ def foreman_org_path(sub_path)
   end
 
   def documentation_url(section = "", options = {})
-    root_url = options[:root_url] || foreman_org_path("manuals/#{SETTINGS[:version].short}/index.html#")
+    version = SETTINGS[:version].tag == 'develop' ? 'nightly' : SETTINGS[:version].short
```

Comment:
We already have 
https://github.com/theforeman/foreman/blob/38b22032e97df4701eb28b3c71ed7f7050230191/app/controllers/links_controller.rb#L63

Time to it to become a real method?

Reply:
I thought about it too and will update.